shell/platform/embedder/embedder.h

```diff
@@ -434,6 +438,55 @@ typedef struct {
   BoolPresentInfoCallback present_with_info;
 } FlutterOpenGLRendererConfig;
 
+/// Alias for id<MTLDevice>.
+typedef const void* FlutterMetalDevice;
+
+/// Alias for id<MTLCommandQueue>.
+typedef const void* FlutterMetalCommandQueue;
+
+/// Alias for id<MTLTexture>.
+typedef const void* FlutterMetalTextureBuffer;
```

Comment:
Nit: Just `FlutterMetalTexture` should be fine and matches the Apple API naming convention.

Reply:
Oh, I see. There is already the struct named that. Can be append `Handle` to these `void *` typedefs?